frontend/src/metabase/admin/databases/components/DeleteDatabaseModal.jsx

```diff
@@ -46,14 +47,11 @@ export default class DeleteDatabaseModal extends Component {
       formError = <span className="text-error px2">{errorMessage}</span>;
     }
 
-    // allow English or localized
-    const confirmed =
-      confirmValue.toUpperCase() === "DELETE" ||
-      confirmValue.toUpperCase() === t`DELETE`;
+    const confirmed = confirmValue === database.name;
 
     return (
       <ModalContent
-        title={t`Delete this database?`}
+        title={`${database.name}: ${t`Delete this database?`}`}
```

Comment:
This doesn't read naturally, and I'd prefer we construct this so it reads:

`Delete the {database.name} database?`

I get that this would require a string change, but that's okay if this is going in to 0.39. If we really want to avoid any string changes (i.e., we want it to go into 0.38 or 0.38.1), then I'd prefer leaving out this change entirely.

Reply:
Alright, updated it.